### PR TITLE
Update build artifact stats

### PR DESCRIPTION
**Pull request by andrewtchin**
Add additional hashes and file size required for GA release data

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -132,6 +132,9 @@ build:
       - cp $BIN/bootstrap.iso $BIN_TEMP_DIR
       - tar czvf $BIN/vic_$$BUILD_NUMBER.tar.gz -C $BIN vic
       - shasum -a 256 $BIN/vic_$$BUILD_NUMBER.tar.gz
+      - shasum -a 1 $BIN/vic_$$BUILD_NUMBER.tar.gz
+      - md5sum $BIN/vic_$$BUILD_NUMBER.tar.gz
+      - du -h $BIN/vic_$$BUILD_NUMBER.tar.gz
```
Comment (mdubya66) [inline .drone.yml]: Any way to get size to 2 decimal places?

so, 230.21M?

Otherwise, LGTM

**Comment by anchal-agrawal** [comment]
LGTM

File: .drone.yml (hunk 1 above)
Comment (mdubya66) [inline .drone.yml]: :) http://superuser.com/questions/633144/du-h-with-more-decimal-places

Comment (mdubya66) [inline .drone.yml]: but we can do without for now. let's keep it simple.

Comment (andrewtchin) [inline .drone.yml]: that didnt work in ubuntu. vlad helped me find a way though :-)

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
lgtm

**Pull request merged**